ci/pipeline.yml

```diff
@@ -91,52 +107,96 @@ resources:
 - name: windows2019-stemcell
   type: bosh-io-stemcell
   source:
-    name: bosh-google-kvm-windows2019-go_agent
+    name: bosh-aws-xen-hvm-windows2019-go_agent
+
+- name: golang-release
+  type: git
+  source:
+    uri: https://github.com/bosh-packages/golang-release.git
 
 jobs:
-- name: bump-cf-test-helpers
+- name: update-cf-smoke-tests
+  serial: true
   public: true
+  serial_groups: [update-repositories]
   plan:
   - in_parallel:
-    - get: cf-smoke-tests
-    - get: runtime-ci
+    - get: golang-release
+      trigger: true
     - get: cf-test-helpers
       trigger: true
-
+    - get: runtime-ci
+    - get: cf-smoke-tests
+  - task: bump-go-deps
+    file: golang-release/ci/tasks/shared/bump-deps.yml
+    input_mapping:
+      input_repo: cf-smoke-tests
+    output_mapping:
+      output_repo: cf-smoke-tests
+    params:
+      SOURCE_PATH: .
   - task: run-cf-test-helpers-unit-tests
     file: runtime-ci/tasks/run-cf-test-helpers-unit-tests/task.yml
-
   - task: bump-cf-test-helpers
     file: runtime-ci/tasks/bump-cf-test-helpers/task.yml
     input_mapping:
       repository: cf-smoke-tests
+    output_mapping:
+      updated-repository: cf-smoke-tests
     params:
       USE_GO_MOD: true
       REPOSITORY: cloudfoundry/cf-smoke-tests
-
   - put: cf-smoke-tests
     params:
-      repository: updated-repository
+      repository: cf-smoke-tests
       rebase: true
 
-- name: bump-cf-smoke-tests
+- name: update-cf-smoke-tests-release
+  serial: true
   public: true
+  serial_groups: [update-repositories]
```

Comment:
Somewhat semantic, but why not call this `update-dependencies`, same as the display group its in above?

Reply:
Yeah fair enough